v2/spiffetls/tlsconfig/config.go

```diff
@@ -89,25 +159,63 @@ func MTLSWebServerConfig(cert *tls.Certificate, bundle x509bundle.Source, author
 // it will be wrapped by by this package and invoked after SPIFFE
 // authentication has completed.
 func HookMTLSWebServerConfig(config *tls.Config, cert *tls.Certificate, bundle x509bundle.Source, authorizer Authorizer) {
-	panic("not implemented")
+	config.InsecureSkipVerify = true
+	config.GetClientCertificate = func(*tls.CertificateRequestInfo) (*tls.Certificate, error) {
+		return cert, nil
+	}
+	config.VerifyPeerCertificate = wrapVerifyPeerCertificate(config.VerifyPeerCertificate, VerifyPeerCertificate(bundle, authorizer))
+
+	// No used by server
```

Comment:
All of these are used by the server. `GetClientCertificate` can go here though once you address the above comment.